include/messageCreate.h

```diff
@@ -18,6 +18,10 @@
  * 
  * ********************************************************/
 
+#ifndef MESSAGE_CREATE_H
+#define MESSAGE_CREATE_H
```

Comment:
I'd like the defines to start with "M2_"
So it should be M2_MESSAGE_CREATE_H
But not import for now :)